spec/javascripts/support/jasmine_helper.rb

```diff
@@ -1,15 +1,15 @@
-# Use this file to set/override Jasmine configuration options
-# You can remove it if you don't need it.
-# This file is loaded *after* jasmine.yml is interpreted.
-#
-# Example: using a different boot file.
-# Jasmine.configure do |config|
-#   config.boot_dir = '/absolute/path/to/boot_dir'
-#   config.boot_files = lambda { ['/absolute/path/to/boot_dir/file.js'] }
-# end
+require "jasmine/runners/selenium"
+require "webdrivers/chromedriver"
 
-# Prevent PhantomJS auto install, uses PhantomJS already on your path.
 Jasmine.configure do |config|
   config.prevent_phantom_js_auto_install = true
```

Comment:
We can get rid of this line as we won't be relying on `PhantomJS` anymore – unless I'm missing something

Reply:
👍